.envrc

```diff
@@ -1,10 +1,7 @@
-export FLINK_LIBS=$PWD/flink_libs
-# TODO: hack to make runner.py work temporarily
 export PYTHONPATH=.
 
 make install-dev install-pre-commit-hook
 
-source .venv/bin/activate
+source ./py/.venv/bin/activate
```

Comment:
I moved the venv back into py because I expect we will have multiple independent packages